.ui/library/flProductItemButtonsWrapper.cpx

```diff
@@ -0,0 +1,123 @@
+{
+	"components": [
+		{
+			"className": ".sf-flexLayout .flProductItemButtonsWrapper .flProductItemButtonsWrapper.inactive",
+			"hiddenComponent": true,
+			"id": "e605-6f16-e23e-ed52",
+			"props": {
+				"children": [
+					"0f0f-b294-521e-2c70",
+					"4839-3c7e-11e2-d820",
+					"ed3d-0432-4f36-bc35",
+					"51b8-11f6-28ae-3a83",
+					"5267-0660-1eda-9e7f"
+				],
+				"name": "flProductItemButtonsWrapper",
+				"parent": "57f4-201f-4bfc-5fc6"
+			},
+			"source": {
+				"page": "__library__"
+			},
+			"type": "FlexLayout",
+			"userProps": {
+				"flexProps": {},
+				"left": 17.33,
```

Comment:
Could you remove user props from this line? @suleymanbasbug

Reply:
OK.